packages/migrate/migrations/svelte-5/migrate.js

```diff
@@ -0,0 +1,123 @@
+import fs from 'node:fs';
+import { Project, ts, Node } from 'ts-morph';
+import { log_on_ts_modification, update_pkg } from '../../utils.js';
+
+export function update_pkg_json() {
+	fs.writeFileSync(
+		'package.json',
+		update_pkg_json_content(fs.readFileSync('package.json', 'utf8'))
+	);
+}
+
+/**
+ * @param {string} content
+ */
+export function update_pkg_json_content(content) {
+	return update_pkg(content, [
+		['svelte', '^5.0.0'],
+		['svelte-check', '^4.0.0'],
+		['svelte-preprocess', '^6.0.0'],
+		['@sveltejs/kit', '^2.5.27'],
```

Comment:
should probably use the latest enhanced-img to get smaller compiled output
```suggestion
		['@sveltejs/enhanced-img', '^0.3.6'],
		['@sveltejs/kit', '^2.5.27'],
```